pkgs/top-level/all-packages.nix

```diff
@@ -6903,6 +6903,10 @@ with pkgs;
 
   leatherman = callPackage ../development/libraries/leatherman { };
 
+  ledit = callPackage ../development/ocaml-modules/ledit {
```

Comment:
You can also move the package in a more relevant directory (e.g., `tools/misc/` or `tools/system/`).